pkgs/development/compilers/cudatoolkit/auto-add-opengl-runpath-hook.sh

```diff
@@ -2,17 +2,21 @@
 # Run addOpenGLRunpath on all dynamically linked, ELF files
 echo "Sourcing auto-add-opengl-runpath-hook"
 
+elfHasDynamicSection() {
+    patchelf --print-rpath "$1" >& /dev/null
```

Comment:
This file uses 2-space tabs. But not worth a big rebuild just to fix that.

Reply:
My bad. Though I guess occasional inconsistencies are expected when there's no automation to enforce the conventions